### PR TITLE
Added optative target to play a card

### PR DESCRIPTION
**Pull request by Masclins**
This solves #76 and makes it able to use add a target variable when playing cards.

### CODE REVIEW

File: backend/test/rules/simultaneous_test.clj [hunk 1]
```diff
@@ -16,12 +16,12 @@
 (defexpect next-play
   ; Stores next-play
   (expect
-    {:player 0 :index 0 :row 0}
+    {:player 0 :index 0 :row 0 :target nil}
```
Comment (kenan-rhoton) [inline backend/test/rules/simultaneous_test.clj]: I feel like this change should'nt be needed, nonexistence of the key should be treated as if it were nil.

Comment (Masclins) [reply backend/test/rules/simultaneous_test.clj]: yet it is stored as `nil`. I'm not sure I understand what you'd expect here.

Comment (kenan-rhoton) [reply backend/test/rules/simultaneous_test.clj]: 🤔

**Comment by Masclins** [comment]
Is there anything that should be changed?

**Comment by kenan-rhoton** [comment]
Yes, my forgetfulness

**Pull request merged**